### PR TITLE
[Security Content] Expand Abbreviated Tags

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Adds a full form of abbreviated terms in tags

### CODE REVIEW

File: rules/ml/discovery_ml_linux_system_information_discovery.toml [hunk 1]
```diff
@@ -28,7 +28,7 @@ name = "Unusual Linux System Information Discovery Activity"
 risk_score = 21
 rule_id = "d4af3a06-1e0a-48ec-b96a-faf2309fae46"
 severity = "low"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML"]
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", ]
```
Comment (imays11) [inline rules/ml/discovery_ml_linux_system_information_discovery.toml:31]:
```suggestion
tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", "Discovery"]
```

File: rules/ml/discovery_ml_linux_system_network_configuration_discovery.toml [hunk 2]
```diff
@@ -28,7 +28,7 @@ name = "Unusual Linux System Network Configuration Discovery"
 risk_score = 21
 rule_id = "f9590f47-6bd5-4a49-bd49-a2f886476fb9"
 severity = "low"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML"]
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", ]
```
Comment (imays11) [inline rules/ml/discovery_ml_linux_system_network_configuration_discovery.toml:31]:
```suggestion
tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", "Discovery"]
```

File: rules/ml/discovery_ml_linux_system_network_connection_discovery.toml [hunk 3]
```diff
@@ -28,7 +28,7 @@ name = "Unusual Linux Network Connection Discovery"
 risk_score = 21
 rule_id = "c28c4d8c-f014-40ef-88b6-79a1d67cd499"
 severity = "low"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML"]
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", ]
```
Comment (imays11) [inline rules/ml/discovery_ml_linux_system_network_connection_discovery.toml:31]:
```suggestion
tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", "Discovery"]
```

File: rules/ml/discovery_ml_linux_system_process_discovery.toml [hunk 4]
```diff
@@ -28,7 +28,7 @@ name = "Unusual Linux Process Discovery Activity"
 risk_score = 21
 rule_id = "5c983105-4681-46c3-9890-0c66d05e776b"
 severity = "low"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML"]
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", ]
```
Comment (imays11) [inline rules/ml/discovery_ml_linux_system_process_discovery.toml:31]:
```suggestion
tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", "Discovery"]
```

File: rules/ml/discovery_ml_linux_system_user_discovery.toml [hunk 5]
```diff
@@ -28,7 +28,7 @@ name = "Unusual Linux System Owner or User Discovery Activity"
 risk_score = 21
 rule_id = "59756272-1998-4b8c-be14-e287035c4d10"
 severity = "low"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML"]
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", ]
```
Comment (imays11) [inline rules/ml/discovery_ml_linux_system_user_discovery.toml:31]:
```suggestion
tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", "Discovery"]
```

File: rules/ml/privilege_escalation_ml_linux_anomalous_sudo_activity.toml [hunk 6]
```diff
@@ -26,7 +26,7 @@ name = "Unusual Sudo Activity"
 risk_score = 21
 rule_id = "1e9fc667-9ff1-4b33-9f40-fefca8537eb0"
 severity = "low"
-tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML"]
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", ]
```
Comment (imays11) [inline rules/ml/privilege_escalation_ml_linux_anomalous_sudo_activity.toml:29]:
```suggestion
tags = ["Elastic", "Host", "Linux", "Threat Detection", "ML", "Machine Learning", "Privilege Escalation"]
```

File: rules/_deprecated/privilege_escalation_gcp_kubernetes_rolebindings_created_or_patched.toml [hunk 7]
```diff
@@ -29,7 +29,7 @@ references = [
 risk_score = 47
 rule_id = "2f0bae2d-bf20-4465-be86-1311addebaa3"
 severity = "medium"
-tags = ["Elastic", "Cloud", "GCP", "Continuous Monitoring", "SecOps", "Configuration Audit"]
+tags = ["Elastic", "Cloud", "GCP", "Google Cloud Platform", "Continuous Monitoring", "SecOps", "Configuration Audit"]
```
Comment (imays11) [inline rules/_deprecated/privilege_escalation_gcp_kubernetes_rolebindings_created_or_patched.toml:32]:
```suggestion
tags = ["Elastic", "Cloud", "GCP", "Google Cloud Platform", "Continuous Monitoring", "SecOps", "Configuration Audit", "Privilege Escalation"]
```

File: rules/_deprecated/exfiltration_rds_snapshot_export.toml [hunk 8]
```diff
@@ -28,7 +28,7 @@ references = ["https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_Sta
 risk_score = 21
 rule_id = "119c8877-8613-416d-a98a-96b6664ee73a5"
 severity = "low"
-tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Asset Visibility"]
+tags = ["Elastic", "Cloud", "AWS", "Amazon Web Services", "Continuous Monitoring", "SecOps", "Asset Visibility"]
```
Comment (imays11) [inline rules/_deprecated/exfiltration_rds_snapshot_export.toml:31]:
```suggestion
tags = ["Elastic", "Cloud", "AWS", "Amazon Web Services", "Continuous Monitoring", "SecOps", "Asset Visibility", "Exfiltration"]
```

**Review by imays11 — COMMENTED** [review]
Nice! I added a few missing MITRE tactic tags.

**Review by imays11 — APPROVED** [review]
👍🏾

**Review by terrancedejesus — APPROVED** [review]
Just need to update the dates. LGTM.

**Comment by w0rk3r** [comment]
Dates :/
I just solved the conflicts and they are gone :pain:

File: rules/_deprecated/exfiltration_rds_snapshot_export.toml [hunk 9]
```diff
@@ -28,7 +28,7 @@ references = ["https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_Sta
 risk_score = 21
 rule_id = "119c8877-8613-416d-a98a-96b6664ee73a5"
 severity = "low"
-tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Asset Visibility"]
+tags = ["Elastic", "Cloud", "AWS", "Amazon Web Services", "Continuous Monitoring", "SecOps", "Asset Visibility", "Exfiltration"]
```
Comment (terrancedejesus) [inline rules/_deprecated/exfiltration_rds_snapshot_export.toml:31]: We shouldn't update deprecated rules because of backporting issues with these.

**Pull request merged**